libs/utils/trace_analysis.py

```diff
@@ -1303,3 +1303,38 @@ def plotClusterFrequencyResidency(self, clusters=None,
 
         self._plotFrequencyResidency(residencies, 'cluster', xmax, pct, active)
 
+    def getContextSwitchCount(self):
```

Comment:
I would suggest to move this function into Trace, likewise we did for the kernel functions profiling API.
```python
Trace::context_switch_df(self)
```
The idea is to use the Trace module to expose all the APIs which returns a "useful" DataFrame. Indeed this module already has a trace.df() method which exposes the Trappy generated DFs, thus it seems a reasonable solution also to host all other APIs which returns some data.

I would also propose to extend the signature to be a bit more generic:
```python
Trace::context_switch_df(self, pids=None, task_names=None, cpus=None)
```
but this can be eventually done in a second patch ;-)